<launch>
    <node name="reposition_base_command" pkg="hdt" type="hdt_base_planning_node" />
</launch>
